# represents all the keypoints for human pose estimation in meters

yolov8/KeyPoint3D[] data

# frame reference
string frame_id